packages/playground/examples/canvas/index.html

```diff
@@ -6,19 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Phasor Canvas Example</title>
     <style>
+      html,
+      body {
+        overscroll-behavior-x: none;
+      }
       #container {
         position: relative;
         margin: auto;
         margin-top: 10px;
         width: 600px;
         height: 400px;
         border: 1px #ddd solid;
+        overflow: hidden;
+      }
+      #hover-rect {
```

Comment:
There is an visible offset the should be addressed. Maybe can try switching to `box-sizing: border-box`?